### DISCUSSION TITLE
GitHub Classroom: "Repository Access Issue" for non-disabled account

### DISCUSSION THREAD

**Discussion by benjavicente** [community/community#121119] [GitHub Education]
### Hi everyone,

We are using GitHub Classroom with the new _Repo Template Fork setup_ for code assigments. It worked perfectly for our first assigment, but in the second one, one student got the error "Repository Access Issue":

<img width="500" alt="Repository Access Issue" src="https://github.com/community/community/assets/62021328/a74bdc38-88e8-4ed8-923a-5a6f3fef4346">

Considering that the student **created a repository in the first assignment**, and **students could create the repository before and after** this problem was reported, we considered a problem with the account of the student. But, for what i could gather, his account is not banned (or has other type of restrictions), so it should be fine.

**¿How can I, a classroom admin, understand, and ideally help to fix, the problem that this student has?**
The message suggests that we, as admins, can do something, but I couldn't find anything.

---

This is related to #72283

**Comment by samus-aran** [comment]
Hi @benjavicente 

Just to confirm that this student is able to continue using other classroom assignments after this error popped up? Can you share the account username? If you don't feel comfortable you can raise a ticket on Education support selecting the option "Classroom".

**Reply by benjavicente** [reply]
We waited for the next assignment to take action with GitHub. For some unknown reason, it now worked. 

Should I submit a ticker anyway? The incontinence was primarily because the student saw an error message that says that we as classroom admin cloud support them, when we didn't have a way to understand the problem.
